easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -130,7 +130,8 @@ def configure_step(self, srcdir=None, builddir=None):
 
         if srcdir is None:
             if self.cfg.get('srcdir', None) is not None:
-                srcdir = self.cfg['srcdir']
+                # Note that the join returns srcdir if it is absolute
+                srcdir = os.path.join(default_srcdir, self.cfg['srcdir'])
```

Comment:
@Flamefire Can you clarify why this is needed?

Reply:
Programs can set `srcdir` and usually do so using a relative path: https://github.com/easybuilders/easybuild-easyconfigs/pull/10300/files#diff-6573a83383dfd6ecdc9fd30fe232478cL76

This path is relative `self.cfg['start_dir']` but if we change the CWD then the below configure command can't find the start_dir.

Reply:
OK, thanks for clarifying!